### ISSUE TITLE
WP fork with Laravel + Themosis

### ISSUE THREAD

**Issue by valle6** [themosis/framework#152]
Do you already think to create an WP folk with Laravel, using Themosis how the plugin system? So developers would can choose between WP + Themosis or WP Laravel fork + Themosis? Have I said something foolish?

**Comment by jlambe** [comment]
I'm not trying to rewrite WordPress using the Laravel framework. Themosis framework is a WordPress stack that provides a structure in order to easily develop and collaborate on WordPress projects.

When running Themosis, you're running a fresh copy of WordPress, so you're still doing "WordPress development" but with shortcuts and extra features. The Themosis framework is a WordPress plugin that provides classes with the same code syntax from Laravel for the WordPress APIs and it's not the Laravel framework loaded behind the scenes.

So basically you only choose if you start a project using default WordPress or by using another code structure with Themosis. But at the end, it's still WordPress.

Hope this clarifies the thing.
